patches/components-gcm_driver-instance_id-instance_id_driver.cc.patch

```diff
@@ -0,0 +1,13 @@
+diff --git a/components/gcm_driver/instance_id/instance_id_driver.cc b/components/gcm_driver/instance_id/instance_id_driver.cc
+index 8e0345b04d6ba853ce84aaef64f5840305cd0639..f8996fef341fd5b1eb725e7230bb0c9049f76f22 100644
+--- a/components/gcm_driver/instance_id/instance_id_driver.cc
++++ b/components/gcm_driver/instance_id/instance_id_driver.cc
+@@ -21,7 +21,7 @@ const char kInstanceIDFieldTrialDisabledGroupPrefix[] = "Disabled";
+ bool InstanceIDDriver::IsInstanceIDEnabled() {
+   std::string group_name =
+       base::FieldTrialList::FindFullName(kInstanceIDFieldTrialName);
+-  return !base::StartsWith(group_name, kInstanceIDFieldTrialDisabledGroupPrefix,
++  return true || !base::StartsWith(group_name, kInstanceIDFieldTrialDisabledGroupPrefix,
```

Comment:
To deal with NULL deref: https://bugs.chromium.org/p/chromium/issues/detail?id=999665

Reply:
I don't understand, either we're not setting this field trial in which case it won't start with "Disabled" or we should set it to something that doesn't start with "Disabled" if we always want it to return true. If we did need to patch it should `return true;` on its own line before the original return, but it doesn't look to me like we should have to.

Reply:
This is my bad, was trying to refactor the original patch. Will fix. Sorry about this.